configs/warnings.tslint.json

```diff
@@ -11,6 +11,40 @@
       "severity": "warning",
       "options": true
     },
+    "no-implicit-dependencies": {
+      "severity": "warning",
+      "options": [
+        true,
+        "dev",
```

Comment:
it should be fine to exclude all dev dependencies.

Reply:
I removed `"dev"`.